install/install.go

```diff
@@ -167,6 +168,28 @@ type AzureParameters struct {
 	ClientSecret         string
 }
 
+var (
+	// DeprecatedFlagsToHelmOpts maps the deprecated install flags to the helm
```

Comment:
nit: Now that these flags are not deprecated anymore, should we name this maybe just `FlagsToHelmOpts`. Or what's your long-term plan for them (also with regards to the TODOs you added)?

Reply:
Renamed. I've left the code commented for now if we later decide to mark them as deprecated.